src/gyazo.rb

```diff
@@ -1,15 +1,26 @@
 #!/usr/bin/env ruby
 
-# setting
-browser_cmd = 'xdg-open'
-clipboard_cmd = 'xclip'
-
 require 'net/http'
 require 'open3'
 require 'openssl'
 require 'json'
 require 'yaml'
 
+# setting
+configfile = "#{ENV['HOME']}/.gyazo.config.yml"
+if File.exist?(configfile) then
+  config = YAML.load_file(configfile)
+end
+
+browser_cmd = config == nil ? 'xdg-open' : config['browser_cmd']
```

Comment:
Yes. I think that your suggestion is good.
I will revise other lines too.
However, when config['use_ssl'] is set in false, can't be distinguished from nil.
Therefore I compare it with nil explicitly. Do you have better idea?